src/pudl/transform/eia.py

```diff
@@ -540,9 +545,9 @@ def harvest_entity_tables(  # noqa: C901
     entity_df = entity_id_df.copy()
     annual_df = annual_id_df.copy()
     special_case_cols = {
-        "latitude": [_lat_long, 1],
-        "longitude": [_lat_long, 1],
-        "generator_operating_date": [_round_operating_date, "Y"],
+        "latitude": {"method": _lat_long, "round_to": 1},
```

Comment:
To do: confirm that I'm not changing the rounding behavior by defaulting to 2 instead of 1

Reply:
0.1 degrees can be as much as 7 miles. Does that feel like it's close enough to consider consistent?

Reply:
I'm not changing the number here, just reformatting how it is fed into the harvesting. If that's a decision we feel unsure about, we should probably take it up in another PR.

Reply:
Verified this works as expected

Reply:
Ah, sorry I misread this somehow as going from rounding to 2 decimals to 1. Disregard!